api/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "api",
+  "version": "1.0.0",
+  "description": "",
+  "main": "dist/index.js",
+  "scripts": {
+    "build": "tsc",
+    "start": "tsc & node .",
```

Comment:
Better use npm run build && node .
But for simpility sake, you can keep start script w/o build operation.